packages/universal-components/src/RangeDatePicker/DraggableItem.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import React from 'react';
+
+// @TODO Add implementations for web and Android
+export default class DraggableDay extends React.Component<{}> {
```

Comment:
Can we name this `DraggableItem` for consistency?

Reply:
Oh, I forgot about this one during the renaming :) thanks